aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -866,6 +1091,10 @@ copy_weights_to_flat_buf_views(
   RNNDescriptorParams rnn;
   rnn.set(
       mode,
+#if defined(CUDNN_VERSION) && CUDNN_VERSION >= RNNV8VERSION
+      input_size,
+      false, // bogus
```

Comment:
Should we do something about it?

Reply:
Added a comment, basically it's a "don't care" value for this function call.